conflux-sponsor-faucet-sdk/faucet-frontend/faucet-frontend.js

```diff
@@ -1,6 +1,7 @@
 const {Conflux} = require('js-conflux-sdk');
 const BigNumber = require('bignumber.js');
-const faucetABI = require('./build/contracts/SponsorFaucet.json');
+const faucet_contract = require('./build/contracts/SponsorFaucet.json');
+const cpc_contract = require('./build/contracts/SponsorWhitelistControl.json');
```

Comment:
what is `cpc`?

suggest to add necessary comments.

Reply:
comments added

Reply:
use **internalSponsorContract** and unify naming style